src/dashboard/luasrc/controller/ids-priv/ids_tab.lua

```diff
@@ -40,7 +40,7 @@ module("luci.controller.ids-priv.ids_tab", package.seeall)  --notice that new_ta
         -- Write JSON file, and make it *pretty*,
         conf:close()
         conf = io.open("/etc/tinyHIPPO/config.json", 'w')
-        conf:write(string.gsub(luci.jsonc.stringify(configuration, true) .. "\n", "\\/", "/"))
+        conf:write((string.gsub(luci.jsonc.stringify(configuration, true) .. "\n", "\\/", "/")))
```

Comment:
Lua is interesting...

Reply:
The fact that adding another set of parentheses solved this issue is mind boggling.

Reply:
Apparently it decouples the values being returned, but intuitively it makes absolutely zero sense